### ISSUE TITLE
Show translations of other languages needing work in Zen

### ISSUE THREAD

**Issue by comradekingu** [WeblateOrg/weblate#6825]
### Describe the problem

I can't fix strings in other languages if I can't see them,
and I can't know they aren't needing work if it isn't evident.

### Describe the solution you'd like

All translations for selected languages shown, with "needing work" checkbox.

**Comment by nijel** [comment]
Are you talking about secondary languages?

**Comment by comradekingu** [comment]
Yes. In one sense they are helper-languages, but in so doing they are also the languages a translator is able to correct.

**Comment by nijel** [comment]
Editing these is one click away, I don't think it's desired to be able to edit them in place.

**Comment by comradekingu** [comment]
It is, but that is beside the point.
They aren't showing up as examples if marked for edit.

**Comment by comradekingu** [comment]
https://hosted.weblate.org/translate/escapepod/android-app/nb_NO/?checksum=9b7dc392df1d0694
has Danish in red for "other languages"
In Zen there is no Danish secondary 

In my regular nb_NO view, with Danish added as a secondary language:
![bilde](https://user-images.githubusercontent.com/13802408/142406424-d1b79498-93d3-4706-aad7-9c8837ef39e1.png)

**Comment by nijel** [comment]
Okay, the secondary languages currently show only translated strings - to help with the translating.

We probably need a toggle to show all strings there as well - for use case when the translator can translate into more languages. What it currently covers is when translator only can understand other languages and seeing other translation is only helpful when it is valid.
